panels/background/Makefile.am

```diff
@@ -11,6 +11,7 @@ AM_CPPFLAGS =						\
 	$(PANEL_CFLAGS)					\
 	$(BACKGROUND_PANEL_CFLAGS)			\
 	-DGNOMELOCALEDIR="\"$(datadir)/locale\""	\
+	-DVENDOR_CONF_FILE=\""$(VENDOR_CONF_FILE)"\"	\
```

Comment:
Since this is apparently being shared by various panels, I guess it's about time to put this define under `PANEL_CFLAGS` (in a separate commit).